extensions/standard-processors/tests/unit/RetryFlowFileTests.cpp

```diff
@@ -86,7 +85,7 @@ class RetryFlowFileTest {
       optional<bool> processor_uuid_matches_flowfile) {
     reInitialize();
 
-    const std::string output_dir = createTempDir(testController_.get());
+    const std::string output_dir = testController_->createTempDirectory();
```

Comment:
This `output_dir` seems to be unused

Reply:
Yes, it is.  Removed it in https://github.com/apache/nifi-minifi-cpp/pull/1139/commits/4aae1742792f939879a50a211fb39877c068925a.